src/MongoDB.Driver.Core/Core/Connections/KeepAliveValues.cs

```diff
@@ -33,7 +33,7 @@ public byte[] ToBytes()
             var bytes = new byte[12];
             BinaryPrimitives.WriteUInt32LittleEndian(new Span<byte>(bytes, 0, 4), OnOff);
             BinaryPrimitives.WriteUInt32LittleEndian(new Span<byte>(bytes, 0, 4), KeepAliveTime);
```

Comment:
This doesn't look right, rather it should be something like:
`BinaryPrimitives.WriteUInt32LittleEndian(new Span<byte>(bytes, 4, 4), KeepAliveTime);`